TESTS/usb_device/basic/USBTester.cpp

```diff
@@ -32,7 +30,6 @@
 #define VENDOR_TEST_CTRL_NONE               3
 #define VENDOR_TEST_CTRL_IN_DELAY           4
 #define VENDOR_TEST_CTRL_OUT_DELAY          5
-#define VENDOR_TEST_CTRL_NONE_DELAY         6
```

Comment:
why is this delay removed? Shall this be a series of commits rather than "enable usb basic" - does not provide a reason for some changes I am commenting now on

Reply:
> Removed the USB endpoint test class unused variable and partially (The greentea test host side implementation is missed on pyusb_basic.py ) implemented USB request control transfer of VENDOR_TEST_CTRL_NONE_DELAY so it can run with the bare metal profile.

This would explain some changes, should be in the commit msg as well.

Reply:
@0xc0170 Done. Please check

Reply:
I don't think it is worth deleting VENDOR_TEST_CTRL_NONE_DELAY (here and in the .py file) because it breaks the defines and it is not the only one not implemented. For example I don't see any code calling VENDOR_TEST_CTRL_IN_STATUS_DELAY or VENDOR_TEST_CTRL_OUT_STATUS_DELAY

Reply:
After discussion with @evedon have decided to remove all unused USB control transfer request type and also partially implemented test.